Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -586,17 +586,19 @@ public void SaveImages(BaseItem item)
         /// <exception cref="ArgumentNullException">
         /// <paramref name="items"/> or <paramref name="cancellationToken"/> is <c>null</c>.
         /// </exception>
-        public void SaveItems(IEnumerable<BaseItem> items, CancellationToken cancellationToken)
+        public void SaveItems(IReadOnlyList<BaseItem> items, CancellationToken cancellationToken)
         {
             ArgumentNullException.ThrowIfNull(items);
 
             cancellationToken.ThrowIfCancellationRequested();
 
             CheckDisposed();
 
-            var tuples = new List<(BaseItem, List<Guid>, BaseItem, string, List<string>)>();
-            foreach (var item in items)
+            var itemsLen = items.Count;
+            var tuples = new ValueTuple<BaseItem, List<Guid>, BaseItem, string, List<string>>[itemsLen];
```

Comment:
Why switch to `ValueTuple` over the shorthand?

Reply:
`new (T, T, T, T)[]` may look confusing with the new `new()` constructors
But I can change it back if you want

Reply:
That is what I assumed- just wanted to make sure I wasn't missing anything. No need to change